docs/development.md

```diff
@@ -0,0 +1,46 @@
+# Development Environment
+
+This document describes the agreed-upon environment for development. For ease of maintenance, it does not aim to be a
+click by click description, and it assumes familiarity with Windows, Linux, and search engines.
+
+The standard environment is:
+1. Windows
+2. Windows Subsystem for Linux 2 (WSL)
+3. Ubuntu latest in WSL
+4. Visual Studio Code (VS Code)
+
+Other configurations are not prohibited, but keep in mind that when problems arise there will be less support from the
+local community.
+
+## Installation
+
+### 1. Windows
+
+Should be installed for you - easy!
+
+### 2. Windows Subsystem for Linux 2 (WSL)
+
+Follow Microsoft's instructions.
+
+### 3. Ubuntu latest in WSL
+
+- Install Ubuntu latest.
+- `sudo apt update`
+- `sudo apt upgrade`
+- Ubuntu 19.10 comes with Python 3.6, so `sudo apt install python3.7 python3.7-dev`
+- Run `python3 -V` and if it's still 3.6.x then update the link to go to 3.7.
+- `sudo apt install python3-pip`
+- `sudo apt install python3.7-venv`
+- Download the oc client tools matching the current version of OpenShift, and put the *oc* binary in your *~/bin*
```

Comment:
I have some steps pasted in a note from awhile ago. If we want to be more explicit.  I guess we would have to maintain the URL. and other stuff.  I am happy to exclude this portion.

```cd /usr/local/bin
sudo wget -c https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
tar -xzf  openshift*.tar.gz  2> /dev/null && rm -rf *.tar.gz && cd open*
mv oc .. 
```